presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestCredentialPassThrough.java

```diff
@@ -25,36 +24,34 @@
 
 import static io.airlift.testing.Closeables.closeAllSuppress;
 import static io.prestosql.testing.TestingSession.testSessionBuilder;
-import static java.lang.String.format;
 
-public class TestCredentialPassthrough
+public class TestCredentialPassThrough
 {
-    private static final String TEST_SCHEMA = "test_database";
     private final TestingMySqlServer mysqlServer;
     private final QueryRunner mySqlQueryRunner;
 
-    public TestCredentialPassthrough()
+    public TestCredentialPassThrough()
             throws Exception
     {
-        mysqlServer = new TestingMySqlServer("testuser", "testpass", TEST_SCHEMA);
+        mysqlServer = new TestingMySqlServer();
         mySqlQueryRunner = createQueryRunner(mysqlServer);
     }
 
     @Test
-    public void testCredentialPassthrough()
+    public void testCredentialPassThrough()
```

Comment:
i think we treat "passthrough" as a single word, so lowercase `t`

Reply:
same in class name

Reply:
Agreed, plus if we were to change it, that should be a separate commit.